ui/src/components/ui/tooltip.tsx

```diff
@@ -26,11 +26,14 @@ function TooltipProvider({
 }
 
 function Tooltip({
+  children,
   ...props
 }: React.ComponentProps<typeof TooltipPrimitive.Root>) {
   return (
     <TooltipProvider>
-      <TooltipPrimitive.Root data-slot='tooltip' {...props} />
+      <TooltipPrimitive.Root data-slot='tooltip' {...props}>
+        {children}
+      </TooltipPrimitive.Root>
```

Comment:
@Etsija @lamppu As this changes a UI component it would be great if one of you could review it.